GAE/src/org/akvo/flow/api/app/DataPointServlet.java

```diff
@@ -120,14 +135,13 @@ private SurveyedLocaleResponse convertToResponse(List<SurveyedLocale> slList,
         resp.setCode(String.valueOf(HttpServletResponse.SC_OK));
         resp.setResultCount(slList.size());
 
-        List<SurveyedLocaleDto> dtoList = getSurveyedLocaleDtosList(slList, surveyGroupId);
+        List<SurveyedLocaleDto> dtoList = getSurveyedLocaleDtosList(slList, surveyId);
 
-        resp.setSurveyedLocaleData(dtoList);
+        resp.setDataPointData(dtoList);
         return resp;
     }
 
-    private List<SurveyedLocaleDto> getSurveyedLocaleDtosList(List<SurveyedLocale> slList,
-            Long surveyGroupId) {
+    protected static List<SurveyedLocaleDto> getSurveyedLocaleDtosList(List<SurveyedLocale> slList, Long surveyId) {
```

Comment:
Just out of curiosity, why is this method static?

Reply:
Ok I understood why, see my comment below why I may suggest to do it differently.